frontend/packages/console-metal3/src/plugin.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  Plugin,
+  ResourceNSNavItem,
+  ResourceListPage,
+  ResourceDetailPage,
+  ModelBasedFeatureFlag
+} from '@console/plugin-sdk';
+
+import { BaremetalHostModel } from './models';
+
+type ConsumedExtensions =
+  | ResourceNSNavItem
+  | ResourceListPage
+  | ResourceDetailPage
+  | ModelBasedFeatureFlag;
+
+const plugin: Plugin<ConsumedExtensions> = [
+  {
+    type: 'FeatureFlag/ModelBased',
+    properties: {
+      model: BaremetalHostModel,
+      flag: 'METAL3'
+    }
+  },
+  {
+    type: 'NavItem/ResourceNS',
+    properties: {
+      section: 'Compute',
```

Comment:
This currently yields a TypeScript error since `ExtensionProperties.NavItem.section` is hard-coded as:

```
'Home' | 'Workloads'
```

I'm working on a PR to fix this.

In the meantime, please update the `packages/console-plugin-sdk/src/typings/nav.ts` module to include the sections you need.

Reply:
cc @jelkosz ^^ is related to the "Improve nav item extension capabilities" task.

Reply:
Done

Reply:
#1584 should fix this.